infrastructure/zcli/src/index.ts

```diff
@@ -151,6 +153,20 @@ async function main() {
             print(commands.listWallets(config));
         });
 
+    wallets
+        .command('generate <network>')
+        .description('generate a new wallet on L2')
+        .action(async (network: Network = 'rinkeby') => {
+            console.log('Using ' + network);
+
+            const ethersProvider = ethers.getDefaultProvider(network);
+            const syncProvider = await zksync.getDefaultProvider(network);
+
+            const ethWallet = ethers.Wallet.createRandom().connect(ethersProvider);
```

Comment:
I think we are always creating a Random wallet here, while it would be great if we can create a wallet from a private key received from command line. And eventually, if the user doesn't specify the L1 private key, we could create a random wallet.

Reply:
Oh yes, this is the old javascript script. I will remove it to just let the rust implementation